### PR TITLE
Adds action for the update command, and instructions. Fixes #41.

### LINKED ISSUE #41: dataset.json does not update when doing a spark-reload

### PR DESCRIPTION
**Pull request by kerchner**

### CODE REVIEW

**Comment by kerchner** [comment]
Recommended test:

- Make sure that your `loader.docker-compose.yml` is pulling from local code, i.e.
```
      #image: gwul/tweetsets-loader
      build:
          context: ..
          dockerfile: Dockerfile-loader
```

- Note an existing collection's dataset metadata as viewed in the UI.

- Modify the collection's `dataset.json` file in its dataset loading folder.

- Run update instruction in this branch's README.

- Refresh the UI, and observe that the dataset metadata is updated with the change you made to `dataset.json`

File: README.md [hunk 1]
```diff
@@ -189,6 +189,34 @@ loader Elasticsearch is on the same machine as TweetSets (e.g., in a small devel
...
+        docker-compose -f loader.docker-compose.yml run --rm loader /bin/bash
+2. Invoke the loader:
+
+        spark-submit \
+        --jars elasticsearch-hadoop.jar \
+        --master spark://$SPARK_MASTER_HOST:7101 \
+        --py-files dist/TweetSets-2.0-py3.6.egg,dependencies.zip \
+        --conf spark.driver.bindAddress=0.0.0.0 \
+        --conf spark.driver.host=$SPARK_DRIVER_HOST \
+        tweetset_loader.py spark-reload dataset-id
```
Comment (lwrubel) [inline README.md]: The spark-reload command also requires a /dataset/path/to argument. This line should be:
`tweetset_loader.py spark-reload dataset-id /dataset/path/to`

**Review by lwrubel — CHANGES_REQUESTED** [review]
Confirmed that the dataset.json updates as expected. Note one edit to the README.

File: README.md (hunk 1 above)
Comment (kerchner) [reply README.md]: @lwrubel path is now added.  Please review.

**Comment by lwrubel** [comment]
Looks good to me!

**Pull request merged**